Automattic/jetpack
# Forms: Response Inbox - Bulk Actions menu

## Proposed changes:

* Replaces the bulk actions dropdown with a bulk actions menu

### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

Enable the jetpack forms package and the new dashboard before testing with:

```
add_filter( 'jetpack_contact_form_use_package', '__return_true' );
add_filter( 'jetpack_forms_dashboard_enable', '__return_true' );
```

- Go to `/wp-admin/admin.php?page=jetpack-forms`
- Use the checkboxes to select responses 
- Play with the bulk actions on top of the table and check if the actions work properly
